transaction/core/test-utils/src/lib.rs

```diff
@@ -96,6 +106,42 @@ pub fn create_transaction_with_amount<L: Ledger, R: RngCore + CryptoRng>(
     fee: u64,
     tombstone_block: BlockIndex,
     rng: &mut R,
+) -> Tx {
+    create_transaction_with_amount_and_comparer(
```

Comment:
What's the tradeoff between creating a new function for this and just including an option in the existing fn "create_transaction_with_amount"?